string rsrp4g
string rsrq4g
string rssi4g
string sinr4g
string pcid
string band4g
